### ISSUE TITLE
Jalla crashes when requiring choo-meta

### ISSUE THREAD

**Issue by auggod** [jallajs/choo-meta#1]
I'm using choo-meta with `Jalla` and it crashes with with the following error after hitting the server.

```
/Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/stream-splicer/index.js:128
        stream.on('error', function (err) {
               ^

TypeError: Cannot read property 'on' of null
    at /Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/stream-splicer/index.js:128:16
    at Labeled.Pipeline.splice (/Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/stream-splicer/index.js:140:7)
    at Labeled.splice (/Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/labeled-stream-splicer/index.js:64:37)
    at Labeled.Pipeline.push (/Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/stream-splicer/index.js:89:17)
    at Labeled.Duplex._destroy (/Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/readable-stream/lib/_stream_duplex.js:127:8)
    at Labeled.destroy (/Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/readable-stream/lib/internal/streams/destroy.js:36:8)
    at /Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/pump/index.js:43:45
    at call (/Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/pump/index.js:50:3)
    at Array.forEach (<anonymous>)
    at /Users/auggod/dev/@auggod/chrome-extensions/bookmark-manager/node_modules/pump/index.js:70:25
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @auggod/bookmark-manager@1.0.0 start: `jalla index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @auggod/bookmark-manager@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

After investigating, i found that it fails at the following line:
https://github.com/jallajs/choo-meta/blob/f18b1abb17afb5d0733786146f8744676d63214d/index.js#L8

Oddly, replacing `state.meta = state.meta || {}` with `state.metadata = state.metadata || {}` for example, makes the problem disappear.
I have no idea why this is happening. Is setting `state.meta` shadowing something at some point? Easiest solution right now would be to set the name of the meta object to something else.

**Comment by tornqvist** [comment]
Good catch! Jalla was naively rendering `<meta />` tags during SSR based on the properties in `state.meta`. But an empty meta object lead to the renderer failing. I just published a fix (`jalla@0.11.1`), updating jalla should resolve the issue you're having.

**Comment by auggod** [comment]
Thanks! Works fine now. I'll close the issue then.
